doc/basics.md

```diff
@@ -2621,6 +2623,120 @@ Performance tips
   ```
 - To better understand the operation of your On Demand parser, and whether it is performing as well as you think it should be, there is a  logger feature built in to simdjson! To use it, define the pre-processor directive `SIMDJSON_VERBOSE_LOGGING` prior to including the `simdjson.h` header, which enables logging in simdjson. Run your code. It may generate a lot of logging output; adding printouts from your application that show each section may be helpful. The log's output will show step-by-step information on state, buffer pointer position, depth, and key retrieval status. Importantly, unless `SIMDJSON_VERBOSE_LOGGING` is defined, logging is entirely disabled and thus carries no overhead.
 
+
```

Comment:
I feel like if possible we should shove this elsewhere--maybe in `performance.md`. It feels out of place for someone who wants to read and know all the basics. Not the biggest deal though.

Reply:
@jkeiser Right. I'll move it to `performance.md`. Next commit.